src/modules/samples/rust/Cargo.lock

```diff
@@ -0,0 +1,7 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Is this supposed to be checked in?

Reply:
Yes, because it just snapshots the version dependencies for the package for replication purposes. 
https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries